docs/ping-identity-oidc-setup.md

```diff
@@ -0,0 +1,50 @@
+# Ping Identity OIDC Authentication Provider Setup
+
+### Prerequisite: Configure Ping Identity Application
+
+Set up the Ping Identity environment and application. Refer to the [Ping Identity Application Setup](./ping-identity-env-setup.md) documentation for instructions.
+
+#### Edit Application Configuration
+
+Navigate to the `Application Overview > Configuration Tab` and click the edit button.
+
+- Check `Code` in `Response Type` and `Authorization Code` in `Grant Type`.
+
+![Response and Grant Type for Ping Identity Application](./images/response_and_grant_type_ping_identity_app.png){ width=30% }
+
+- Set Redirect URIs to `https://your-backstage.com/api/auth/oidc/handler/frame`.
+
+Navigate to the `Application Overview > Resources Tab` and click the edit button.
+
+- Add `email`, `offline_access` and `profile` to allowed scopes.
+
+![Allowed Scopes for Ping Identity Application](./images/allowed_scopes_ping_identity_app.png){ width=50% }
+
+**Note:** the display name in the RHDH instance maps to `Formatted Name` in Ping Identity user profile. If this parameter is not set, it will default to the user entity ref.
```

Comment:
Maybe show a screenshot of where to set this parameter?

Reply:
I added some instructions on how to set this, in this case a screenshot isn't super helpful :)